pkg/ipamd/ipamd.go

```diff
@@ -2188,3 +2221,29 @@ func (c *IPAMContext) isConfigValid() bool {
 
 	return true
 }
+
+func (c *IPAMContext) BroadcastMissingPermissionsEvent(nodeInitErr error) {
+	log.Errorf("Failed to start aws-node due to missing policy, verify AmazonEKS_CNI_Policy(or equivalent) is attached to supplied IAM Role, err:%v", nodeInitErr)
```

Comment:
`AmazonEKS_CNI_Policy` doesn't apply for v6 clusters though and in addition we now support cluster scoped policy. We should just point them to refer to our docs - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/iam-policy.md